BiblioPixelSerialReceiver.ino

```diff
@@ -7,13 +7,13 @@ https://github.com/ManiacalLabs/BiblioPixelSmartMatrix
 **********************************/
 
 #include "FastLED.h"
-#include <EEPROM.h>
+//#include <EEPROM.h> // not available in M0
 
 /****************************
 All Firmware options go here!
 ****************************/
 // How many leds in your strip?
-#define NUM_LEDS 8 * 8
+#define NUM_LEDS 32 * 56
```

Comment:
These should stay just something basic for this repo. You can change them in your local usage.